dot/rpc/http_test.go

```diff
@@ -90,3 +100,214 @@ func TestNewHTTPServer(t *testing.T) {
 
 	require.Equal(t, "405 Method Not Allowed", res.Status)
 }
+
+func TestUnsafeRPCProtection(t *testing.T) {
+	cfg := &HTTPServerConfig{
+		Modules: []string{"author"},
+		RPCPort: 7878,
+		RPCAPI:  NewService(),
+	}
+
+	s := NewHTTPServer(cfg)
+	err := s.Start()
+	require.NoError(t, err)
+
+	time.Sleep(time.Second)
+
+	for _, unsafe := range modules.UnsafeMethods {
+		func(method string) {
```

Comment:
Remove this function. This is not needed.

Reply:
done!